docs/adr/014-rollover-2.0.md

```diff
@@ -0,0 +1,91 @@
+# ADR 014 - Rollover 2.0
+
+```
+Author: @thomaseizinger
+Date: 20/01/2022
+Status: Active
+```
+
+## Summary
+
+Revise the design of the rollover protocol to:
+
+1. Run at fixed points in time.
+2. Not require approval by either party.
+
+## Context
+
+Currently, rollover is a protocol that is initiated by the taker pretty much on the hour every hour.
+The exact conditions for triggering are that we want the timestamp of the price event that is used for attestation to be 24h in the future.
+Because the oracle attests on the price exactly every hour, we can rollover to a new price event on the hour.
+
+Rollover is coupled to paying fees.
+In particular, the time window until the attestation can be considered as the current funding period.
+For every new funding period, fees are charged depending on the current market.
+In a bull market, longs pay shorts.
+In a bear market, shorts pay longs.
+
+To harmonize, how these fees are paid and where they are coming from, we want to revise the rollover protocol as per the design below.
+
+## The new protocol
+
+### 1. We define fixed UTC timestamps for the funding periods
+   
+The exact timestamps are not decided yet, but they will be something along the lines of:
+   - Every day at 0400 UTC.
+   - Every day at 0800 UTC.
+   - Every day at 1200 UTC.
+   - Every day at 1600 UTC.
+   - Every day at 2000 UTC.
+   - Every day at 0000 UTC.
+
+Running rollover at exactly these points makes implementations easier because they can assume that well-behaved peer will execute the protocol at these points in time.
+
+It makes UI design easier because we can clearly state, when the next funding event happens, and it does not depend on the maker that we are connected to.
+
+Another benefit of fixed funding periods is that our oracle actor only needs to fetch attestations for specific points in time and not every hour.
```

Comment:
For our _dynamic liquidation_ feature we will need to fetch more regularly.

Reply:
True, not a concern of this protocol though so at when viewed in isolation, this is still true!